components/auth/Register.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, FC } from 'react';
+import styled from 'styled-components';
+import Section from '@/components/Section';
+import { media } from '@/utils/theme';
+
+import RegisterInputName from '@/components/auth/RegisterInputName';
+import RegisterSignIn from '@/components/auth/RegisterSignIn';
+import FinshPage from '@/components/auth/FinshPage';
+// import RegisterEmail from '@/components/auth/RegisterEmail';
+// import RegisterPhone from '@/components/auth/RegisterPhone';
+// import RegisterEmailVerify from '@/components/auth/RegisterEmailVerify';
+// import RegisterPhoneVerify from '@/components/auth/RegisterPhoneVerify';
+// ----all pageState component----- //
+
+
+const StyledSection = styled(Section)`
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 100vh;
+  > img {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+const RegisterWrapper = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  ${media('desktop')} {
+    width: 100%;
+  }
+`;
+
+const routes: any = {
+  // ['RegisterEmail']: RegisterEmail,
+  // ['RegisterPhone']: RegisterPhone,
+  // ['RegisterEmailVerify']: RegisterEmailVerify,
+  // ['RegisterPhoneVerify']: RegisterPhoneVerify,
```

Comment:
If we don't need these components now, please leave a comment and try explain it for other people~